languages/csharp/AppEncryption/README.md

```diff
@@ -197,22 +71,19 @@ AWSConfigs.AWSRegion = "us-west-2";
 
 // Build the DynamoDB Metastore.
 IMetastorePersistence<JObject> dynamoDbMetastorePersistence = DynamoDbMetastorePersistenceImpl.NewBuilder().Build();
+```
 
-// Use the Metastore for the session factory
-using (AppEncryptionSessionFactory appEncryptionSessionFactory = AppEncryptionSessionFactory.NewBuilder("productId", "systemId")
-     .WithMetastorePersistence(dynamoDbMetastorePersistence)
-     .WithCryptoPolicy(policy)
-     .WithKeyManagementService(keyManagementService)
-     .Build()) {
+#### In-memory Metastore (FOR TESTING ONLY)
 
-    // ...
-}
+```c#
+MetastorePersistence<JObject> metastorePersistence = new MemoryPersistenceImpl<>();
```

Comment:
fix syntax errors here (interface name and can't use java diamond operator inference)